doc/modules/outlier_detection.rst

```diff
@@ -236,20 +289,22 @@ where abnormal samples have different underlying densities.
 The question is not, how isolated the sample is, but how isolated it is
 with respect to the surrounding neighborhood.
 
+When applying LOF for outlier detection, there are no ``predict``,
+``decision_function`` and ``score_samples`` methods but only a ``fit_predict`` method. The scores of abnormality of the training samples are accessible
+through the ``negative_outlier_factor_`` attribute.
```

Comment:
Add a not that this is however possible when setting the novelty parameter to True. And put a cross-ref to the section that explains this.